### PR TITLE
Improve URL redaction

### PR DESCRIPTION
**Pull request by adambkaplan**
Addresses issue brought up in BZ 1571349

### CODE REVIEW

File: pkg/util/env_test.go [hunk 1]
```diff
@@ -4,60 +4,54 @@ import (
...
+	expected := make([]string, len(cases))
+	for i, c := range cases {
+		inputs[i] = c.Input
+		expected[i] = c.Output
 	}
 	result := SafeForLoggingEnv(inputs)
 	for i := range result {
 		if result[i] != expected[i] {
-			t.Errorf("expected %s to be stripped to %s, but got %s", inputs[i], expected[i], result[i])
+			t.Errorf("expected %s to be changed to %s, but got %s", inputs[i], expected[i], result[i])
```
Comment (bparees) [inline pkg/util/env_test.go]: `expected %s to be %s`  (this test no longer expects the string to be changed in all cases, there are some cases where it should be untouched)

**Comment by adambkaplan** [comment]
@bparees @gabemontero updated unit test message.

**Comment by bparees** [comment]
[merge]

**Pull request merged**